packages/runtime/agent-scheduler/src/scheduler.ts

```diff
@@ -44,6 +44,7 @@ class AgentScheduler extends EventEmitter implements IAgentScheduler {
         } else {
             root = await runtime.getChannel("root") as ISharedMap;
             const handle = await root.wait<IFluidHandle<ConsensusRegisterCollection<string | null>>>("scheduler");
+            assert(handle !== undefined);
```

Comment:
nit - I do not think asserts (without messages) add any value if we are about to crash on next line. We have moved away from such practice in Office (devmain) over time due to concerns of extra binary size without any gain. JS is a bit different (as stack quite often is garbage), but maybe if we put an effort - we should put a message?

Reply:
It is not needed, I will remove it.